packages/field-label/src/FieldLabel.ts

```diff
@@ -134,19 +134,19 @@ export class FieldLabel extends SizedMixin(SpectrumElement) {
 
     protected firstUpdated(changes: PropertyValues): void {
         super.firstUpdated(changes);
+        this.addEventListener('click', this.handleClick);
+    }
+
+    protected update(changes: PropertyValues): void {
```

Comment:
ditto re: `willUpdate`

Reply:
Asking these two together is a great question. And, while I didn't realize that `willUpdate()` was an option, I have a follow up question. Because Field Label mixes the Sized functionality into itself, there would be a need to call `super` for our code, but not for the Lit code. Would the use of `willUpdate()` and the differences between Lit and SWC point to the need/benefit of more advanced lifecycle documentation on our part? Does using `update()` protect us from the need for that additional documentation? I was about to make this update, until I realized that already we'd be hitting this problem and thought it would be good to have a path forward marked out for this internally.

Reply:
Hm, that's a good question. Maybe things that are expected to be extended, like mixins and SpectrumElement, _should_ use update() since that parallels what Lit does & means the Lit lifecycle recommendations still apply to the final subclasses.

If we went that route, then we'd want to keep `update` on the sized mixin (mirroring Lit) and use `willUpdate` on FieldLabel (like a typical Lit consumer).

Reply:
I think that makes good sense as a rule to support these contexts! Will update.